lib/app/utils/fhir_models_validator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Inferno
+  # FHIRModelsValidator extends BaseValidator to use the validation in fhir_models.
+  # It passes the validation off to the correct model version.
+  class FHIRModelsValidator
+    def initialize; end
+
+    def validate(resource, _fhir_version, profile_url = nil)
```

Comment:
This method signature doesn't match the one in the hl7 validator:
```
def validate(resource, fhir_models_klass, profile = nil)
```

Reply:
Good catch. Will fix.